sdk/deviceupdate/azure-iot-deviceupdate/azure/iot/deviceupdate/aio/_device_update_client.py

```diff
@@ -6,80 +6,89 @@
-    :type credential: ~azure.core.credentials_async.AsyncTokenCredential
-    :param account_endpoint: Account endpoint.
-    :type account_endpoint: str
+    :ivar device_update: DeviceUpdateOperations operations
+    :vartype device_update: azure.iot.deviceupdate.aio.operations.DeviceUpdateOperations
+    :ivar device_management: DeviceManagementOperations operations
+    :vartype device_management: azure.iot.deviceupdate.aio.operations.DeviceManagementOperations
+    :param endpoint: Account endpoint.
+    :type endpoint: str
     :param instance_id: Account instance identifier.
     :type instance_id: str
+    :param credential: Credential needed for the client to connect to Azure.
+    :type credential: ~azure.core.credentials_async.AsyncTokenCredential
+    :keyword api_version: Api Version. The default value is "2021-06-01-preview". Note that
+     overriding this default value may result in unsupported behavior.
+    :paramtype api_version: str
+    :keyword int polling_interval: Default waiting time between two polls for LRO operations if no
+     Retry-After header is present.
     """
 
     def __init__(
         self,
-        credential: "AsyncTokenCredential",
-        account_endpoint: str,
+        endpoint: str,
         instance_id: str,
+        credential: "AsyncTokenCredential",
         **kwargs: Any
     ) -> None:
-        base_url = 'https://{accountEndpoint}'
-        self._config = DeviceUpdateClientConfiguration(credential, account_endpoint, instance_id, **kwargs)
-        self._client = AsyncPipelineClient(base_url=base_url, config=self._config, **kwargs)
+        _endpoint = 'https://{endpoint}'
+        self._config = DeviceUpdateClientConfiguration(endpoint=endpoint, instance_id=instance_id, credential=credential, **kwargs)
+        self._client = AsyncPipelineClient(base_url=_endpoint, config=self._config, **kwargs)
 
-        client_models = {k: v for k, v in models.__dict__.items() if isinstance(v, type)}
-        self._serialize = Serializer(client_models)
+        self._serialize = Serializer()
+        self._deserialize = Deserializer()
         self._serialize.client_side_validation = False
-        self._deserialize = Deserializer(client_models)
+        self.device_update = DeviceUpdateOperations(self._client, self._config, self._serialize, self._deserialize)
+        self.device_management = DeviceManagementOperations(self._client, self._config, self._serialize, self._deserialize)
 
-        self.updates = UpdatesOperations(
-            self._client, self._config, self._serialize, self._deserialize)
-        self.devices = DevicesOperations(
-            self._client, self._config, self._serialize, self._deserialize)
-        self.deployments = DeploymentsOperations(
-            self._client, self._config, self._serialize, self._deserialize)
 
-    async def _send_request(self, http_request: HttpRequest, **kwargs: Any) -> AsyncHttpResponse:
+    def send_request(
```

Comment:
It seems we're also adding `send_request` onto the client for this release, can you add this to the changelog? (unless you don't mean to add send_request)